File: decompiled/sources/com/streamax/client/AlarmInfo.java
package com.streamax.client;

import android.util.Log;

public class AlarmInfo {
    String alarmChannel = "";
    String alarmContent = "";
    String alarmSubType = "";
    String alarmTime = "";
    String alarmType = "";

    /* access modifiers changed from: package-private */
    public void Print(String str) {
        Log.v(str, "**************AlarmInfo start*******************");
        Log.v(str, "alarmTime = " + this.alarmTime);
        Log.v(str, "alarmType = " + this.alarmType);
        Log.v(str, "alarmSubType = " + this.alarmSubType);
        Log.v(str, "alarmChannel = " + this.alarmChannel);
        Log.v(str, "alarmContent = " + this.alarmContent);
        Log.v(str, "**************AlarmInfo end*******************");
    }
}

File: decompiled/sources/com/streamax/client/AlarmInfo.kt
package com.streamax.client

import android.util.Log

class AlarmInfo {
    @JvmField var alarmChannel: String = ""
    @JvmField var alarmContent: String = ""
    @JvmField var alarmSubType: String = ""
    @JvmField var alarmTime: String = ""
    @JvmField var alarmType: String = ""

    fun Print(tag: String) {
        Log.v(tag, "**************AlarmInfo start*******************")
        Log.v(tag, "alarmTime = $alarmTime")
        Log.v(tag, "alarmType = $alarmType")
        Log.v(tag, "alarmSubType = $alarmSubType")
        Log.v(tag, "alarmChannel = $alarmChannel")
        Log.v(tag, "alarmContent = $alarmContent")
        Log.v(tag, "**************AlarmInfo end*******************")
    }
}
